spec/component_guide/component_guide_spec.rb

```diff
@@ -111,4 +111,20 @@
       end
     end
   end
+
+  it 'loads a preview all page for a component' do
+    visit '/component-guide/test-component-with-params/preview'
+    expect(page).to have_selector('.hide-header-and-footer')
+    expect(page).to have_selector('.component-guide-preview-page .preview-title', text: 'Default')
+    expect(page).to have_selector('.component-guide-preview-page .test-component-with-params', text: 'Some value')
+    expect(page).to have_selector('.component-guide-preview-page .preview-title', text: 'Another fixture')
+    expect(page).to have_selector('.component-guide-preview-page .test-component-with-params', text: 'A different value')
+  end
+
+  it 'loads a preview page for a specific fixture' do
+    visit '/component-guide/test-component-with-params/another_fixture/preview'
+    expect(page).to have_selector('.hide-header-and-footer')
```

Comment:
We can use a feature of Capybara to confirm that the header and footer are hidden:

I _think_ we can do something like, even though that element is present because it's hidden Capybara won't return it and this test will pass:
```
expect(page).not_to have_selector('#header')
```